functions/src/constants.ts

```diff
@@ -10,3 +10,81 @@ export type Term =
   | '202205';
 
 // add some code?
```

Comment:
is this a TODO?

Reply:
Uh that has always been there and unrelated to this...

Reply:
ya I was just wondering